src/components/ReplyComment/index.js

```diff
@@ -113,7 +119,7 @@ const ReplyCommentId = ({
                     <ConnectorWrapper index={index}>
                       <Pressable
                         onLongPress={() => {
-                          showAlertDelete(itemReply, false, () => getThisComment(true));
+                          showAlertDelete(itemReply, false, updateComment);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `showAlertDelete` function now takes `updateComment` as an argument instead of `() => getThisComment(true)`. Ensure that `updateComment` is correctly implemented in `showAlertDelete`.
```diff
-                          showAlertDelete(itemReply, false, updateComment);
+                          showAlertDelete(itemReply, false, updateComment); // Ensure updateComment is correctly implemented in showAlertDelete
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->